### PR TITLE
Updates ro locale for v2.x

### PR DESCRIPTION
**Pull request by aocneanu**

### CODE REVIEW

File: src/locale/ro/index.js [hunk 1]
```diff
@@ -11,19 +11,19 @@
  * @language Romanian
  * @iso-639-2 ron
  * @author Sergiu Munteanu [@jsergiu]{@link https://github.com/jsergiu}
+ * @author Adrian Ocneanu [@aocneanu]{@link https://github.com/aocneanu}
+ * @author Mihai Ocneanu [@aocneanu]{@link https://github.com/gandesc}
```
Comment (kossnocorp) [inline src/locale/ro/index.js]: Please fix the link to the author:

```suggestion
 * @author Mihai Ocneanu [@gandesc]{@link https://github.com/gandesc}
```

File: src/locale/ro/test.js [hunk 2]
```diff
@@ -1,9 +1,448 @@
...
-describe.skip('ro locale', function () {
+import format from '../../format'
+import formatDistance from '../../formatDistance'
+import formatDistanceStrict from '../../formatDistanceStrict'
+import formatRelative from '../../formatRelative'
+import parse from '../../parse'
 
+describe('ro locale', function () {
+    context('with `format`', function () {
+        var date = new Date(1986, 3 /* Apr */, 5, 10, 32, 0, 900)
```
Comment (kossnocorp) [inline src/locale/ro/test.js]: Please format this file using [Prettier](https://prettier.io).

**Comment by kossnocorp** [comment]
Thank you a lot!

**Comment by aocneanu** [comment]
No, really, than you and the team for this great lib!

**Pull request merged**